Brain/brain_utils.h

```diff
@@ -39,18 +40,22 @@ struct valveInfo {
   int id;
   int (*openValve)();
   int (*closeValve)();
-  //valveInfo(String n, int v): valve_name(n), valve_id(v) {}
 };
 
 valveInfo valve_ids[7] = {
-  {"LOX 2 Way", 20, &(Solenoids::armLOX), &(Solenoids::disarmLOX)} //example
+  {"LOX 2 Way", 20, &(Solenoids::armLOX), &(Solenoids::disarmLOX)}, //example
+  {"LOX 5 Way", 21, &(Solenoids::openLOX), &(Solenoids::closeLOX)},
+  {"LOX GEMS", 22, &(Solenoids::ventLOXGems), &(Solenoids::closeLOXGems)},
+  {"Propane 2 Way", 23, &(Solenoids::armPropane), &(Solenoids::disarmPropane)},
+  {"Propane 5 Way", 24, &(Solenoids::openPropane), &(Solenoids::closePropane)},
+  {"Propane GEMS", 25, &(Solenoids::ventPropaneGems), &(Solenoids::closePropaneGems)},
+  {"High Pressure Solenoid", 26, &(Solenoids::activateHighPressureSolenoid), &(Solenoids::deactivateHighPressureSolenoid)}
```

Comment:
I think this is a mistake. don't want these in master. Just want the first one

Reply:
Oops thought I removed these, will update